### DISCUSSION TITLE
Mac Users: Suggestions for using BORIS

### DISCUSSION THREAD

**Discussion by shlingle** [olivierfriard/BORIS#648] [Q&A]
Mac Users: I’d like to know what Mac users are doing so as to use BORIS? Can v.13.9 be used on Monterey (OS 12)? Or are you continuing to use an older OS so you can use BORIS? Or are you finding a way to use the Windows or Linus so you can use v.8. We are resuming use of BORIS after a two year gap, and don’t know the best way forward. Thanks!

**Comment by olivierfriard** [comment]
Hi,
my advice is to find a way to use the latest release of BORIS and no longer the v. 7.13.9 which contains bugs and is no longer supported.
See http://www.boris.unito.it/pages/download_mac for running the last version on your Mac.

If you choose to use Docker the script start_macOS.command in the scripts directory should help you.

Olivier

**Reply by shlingle** [reply]
Olivier - Thanks for taking the time to respond!

**Comment by olivierfriard** [comment]
Hi,
I just released a version of BORIS that should run natively on macOS with some limitations described on https://www.boris.unito.it/download_mac.
This is an experimental version so test it and let me know
